templates/MODULE.bazel.inja

```diff
@@ -60,15 +60,15 @@ bazel_dep(name = "rules_shell", version = "0.4.0", dev_dependency = True)
 # Python dependencies
 # ===================
 
-bazel_dep(name = "rules_python", version = "1.5.4")
+bazel_dep(name = "rules_python", version = "1.6.0")
 
 PYTHON_VERSIONS = [
     "3.9",
     "3.10",
     "3.11",
     "3.12",
     "3.13",
-    "3.14.0b2",
+    "3.14",
```

Comment:
Does this work? `rules_python` 1.6.0 only provides `3.14.0rc1`: https://rules-python.readthedocs.io/en/latest/changelog.html#v1-6-0 

IRRC we needed to specify the exact patch version including `.0b2` because `rules_python` doesn't have a stable-release toolchain. @sreenithi - could you confirm?